static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx

```diff
@@ -82,6 +93,17 @@ function ResourceSummary() {
               </PageFilterBar>
             </PaddedContainer>
             <BlockContainer>
+              <Block title={t('Avg encoded size')}>
+                <FileSize bytes={spanMetrics?.[`avg(${HTTP_RESPONSE_CONTENT_LENGTH})`]} />
+              </Block>
+              <Block title={t('Avg decoded size')}>
+                <FileSize
+                  bytes={spanMetrics?.[`avg(${HTTP_DECODED_RESPONSE_BODY_LENGTH})`]}
+                />
+              </Block>
+              <Block title={t('Avg transfer size')}>
```

Comment:
Let's add tooltips via the question mark symbol to explain what these mean.

Reply:
Great idea 👍
Will address in another PR